evaluation/swe_bench/scripts/docker/README.md

```diff
@@ -0,0 +1,13 @@
+# Docker Build Guide
```

Comment:
Do we need `v1.0` and `full_deps`? It seems like they are not used in this README.

Reply:
Yes, we do! `full_deps` contains all the pre-build conda environments and testbed. And `v1.0` build on top of `full_deps` to add the latest version of our code -- and this is the actual image people uses for inference & evaluation.